settings.js

```diff
@@ -1,11 +1,13 @@
-// DON'T FORGET TO ADJUST YOUR SAVED SEARCH SETTINGS, AND SET A SAVED MESSAGE! ./README.md
+// DON'T FORGET TO ADJUST YOUR SAVED SEARCH SETTINGS, AND SET A SAVED MESSAGE IN message.txt!
 
 export default {
-    USERNAME: "", // kamernet email
-    PASSWORD: "", // kamernet password
+    USERNAME: "a@b.com", // kamernet email
+    PASSWORD: "*******",      // kamernet password
     CITY: "delft", // the city name you are looking for (this should be in the url when you perform a search
```

Comment:
maybe write that it should be in lowercase?

Reply:
Case does not matter, kamernet simply redirects the page to the lovercase version. In the code it replaces whitespaces with dashes, but that would be a good idea to lowercase the text too to get rid of that redirect.